invisible_cities/reco/icaro_components.py

```diff
@@ -0,0 +1,201 @@
+import numpy  as np
+import pandas as pd
+
+from   typing               import Tuple, Optional
+from   sklearn.linear_model import RANSACRegressor
+
+from .  corrections         import ASectorMap
+from .  corrections         import apply_geo_correction
+
+from .. types.symbols       import type_of_signal
+from .. types.symbols       import Strictness
+from .. types.symbols       import NormStrategy
+from .. core.core_functions import all_in_range
+from .. core.core_functions import in_range
+from .. core.core_functions import shift_to_bin_centers
+from .. core.fit_functions  import fit
+from .. core.fit_functions  import gauss
+
+def selection_nS_mask_and_checking(dst        : pd.DataFrame                         ,
```

Comment:
```suggestion
def select_nS_mask_and_check(dst        : pd.DataFrame                         ,
```

function names in imperative